<launch>
  <arg name="positions_file" default="/home/roomac/roomac_data/positions.yaml" />

  <node name="pick_and_bring_manager" pkg="roomac_bringup" type="pick_and_bring_manager_node.py" output="screen">
    <rosparam command="load" file="$(find roomac_bringup)/config/pick_and_bring_manager_config.yaml" subst_value="true" />
  </node>
</launch>